test/test_helper.rb

```diff
@@ -1,6 +1,11 @@
+require "simplecov"
+SimpleCov.start('rails')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.